<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>itsadn</author>
 <class>interface_</class>
 <widget class="QMainWindow" name="interface_">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1661</width>
    <height>1095</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1661</width>
    <height>1012</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Poppins</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Calcicité Plagioclase par Fluorescence X Portative</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" rowspan="2" colspan="2">
     <widget class="QFrame" name="frame_files_imported">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QListWidget" name="lw_importedFiles">
         <property name="movement">
          <enum>QListView::Static</enum>
         </property>
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="viewMode">
          <enum>QListView::ListMode</enum>
         </property>
         <property name="batchSize">
          <number>5</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="4" rowspan="8">
     <widget class="QFrame" name="frame_graphique">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGraphicsView" name="gv_chart">
         <property name="cursor" stdset="0">
          <cursorShape>CrossCursor</cursorShape>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="0" rowspan="3" colspan="2">
     <widget class="QGroupBox" name="gb_validation">
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>10</pointsize>
        <stylestrategy>PreferAntialias</stylestrategy>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="title">
       <string>Validation</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_23">
       <item>
        <widget class="QCheckBox" name="cb_probeReference">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Données de Reférence - Microsonde</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>img/xlsx_.ico</normaloff>img/xlsx_.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_pxrfCalibration">
         <property name="text">
          <string>Données de Calibration - pXRF</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>img/xlsx_.ico</normaloff>img/xlsx_.ico</iconset>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_unknowAnalysis">
         <property name="text">
          <string>Données d'analyses inconnu - pXRF</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>img/xlsx_.ico</normaloff>img/xlsx_.ico</iconset>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QGroupBox" name="gb_traitment">
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>10</pointsize>
        <stylestrategy>PreferAntialias</stylestrategy>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="title">
       <string>Traitement</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QPushButton" name="pb_calibrer">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>255</width>
           <height>35</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Calibrer</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>img/gears.ico</normaloff>img/gears.ico</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_start">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>255</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Poppins</family>
           <pointsize>11</pointsize>
           <bold>true</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(44, 202, 57);
background-color: rgb(85, 255, 127);</string>
         </property>
         <property name="text">
          <string>LANCER LE TRAITEMENT</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>img/start.ico</normaloff>img/start.ico</iconset>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QFrame" name="frame_file_imported_title_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="l_procedures">
         <property name="font">
          <font>
           <family>Poppins</family>
           <pointsize>12</pointsize>
           <bold>false</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="text">
          <string>PROCÉDURES</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QFrame" name="frame_file_imported_title_3">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_22">
       <item>
        <widget class="QLabel" name="l_general">
         <property name="minimumSize">
          <size>
           <width>255</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Poppins</family>
           <pointsize>12</pointsize>
           <bold>false</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="text">
          <string>GÉNÉRAL</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="11" column="0" colspan="2">
     <widget class="QGroupBox" name="gb_project">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>10</pointsize>
        <stylestrategy>PreferAntialias</stylestrategy>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="title">
       <string>Projets</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QPushButton" name="pb_openProject">
         <property name="minimumSize">
          <size>
           <width>255</width>
           <height>25</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>ForbiddenCursor</cursorShape>
         </property>
         <property name="text">
          <string>Ouvrir un projet</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pb_saveProject">
         <property name="minimumSize">
          <size>
           <width>255</width>
           <height>25</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>ForbiddenCursor</cursorShape>
         </property>
         <property name="text">
          <string>Enregistrer</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pb_newProject">
         <property name="minimumSize">
          <size>
           <width>255</width>
           <height>25</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>ForbiddenCursor</cursorShape>
         </property>
         <property name="text">
          <string>Nouveau projet</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QFrame" name="frame_file_imported_title">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="l_importedFile">
         <property name="font">
          <font>
           <family>Poppins</family>
           <pointsize>12</pointsize>
           <bold>false</bold>
           <underline>false</underline>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="text">
          <string>FICHIERS IMPORTÉS</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QGroupBox" name="gb_calibration">
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>10</pointsize>
        <stylestrategy>PreferAntialias</stylestrategy>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="title">
       <string>Calibration</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QPushButton" name="pb_formatProbe">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>255</width>
           <height>35</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Format -microsonde</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>img/format.ico</normaloff>img/format.ico</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_formatPXRF">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>255</width>
           <height>35</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Format - pXRF</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>img/format.ico</normaloff>img/format.ico</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="12" column="0" colspan="2">
     <widget class="QGroupBox" name="gb_resultCalcicite">
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>10</pointsize>
        <stylestrategy>PreferAntialias</stylestrategy>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="title">
       <string>Résultat calcicité</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <item>
        <widget class="QPushButton" name="pb_extractResult">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>255</width>
           <height>35</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Extraire - Excel</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>img/save.ico</normaloff>img/save.ico</iconset>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QFrame" name="frame_graphique_title">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_graphique_title">
         <property name="font">
          <font>
           <family>Poppins</family>
           <pointsize>24</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>CALCICITÉ PLAGIOCLASE PAR FLUORESCENCE X PORTATIVE</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="3" rowspan="12">
     <widget class="Line" name="verticalLine_divider">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="9" column="4" rowspan="4">
     <widget class="QFrame" name="frame_tabs">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTabWidget" name="tw_tabs">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>390</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Poppins</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_calibration_probe">
          <attribute name="title">
           <string>Calibration Microsonde</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <widget class="QTableWidget" name="table_probeCalibration"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_calibration_pxrf">
          <attribute name="title">
           <string>Calibration pXRF</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QTableWidget" name="table_calibrationPXRF"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_correction_factor">
          <attribute name="title">
           <string>Facteur de correction</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <item>
            <widget class="QTableWidget" name="table_correctionFactor"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_pxrf_analysis">
          <attribute name="title">
           <string>Analyses pXRF</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_14">
           <item>
            <widget class="QTableWidget" name="table_pxrfAnalysis"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_results">
          <attribute name="title">
           <string>Résultats</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="QTableWidget" name="table_resultAnContent"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QGroupBox" name="gb_importFile">
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>10</pointsize>
        <stylestrategy>PreferAntialias</stylestrategy>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="title">
       <string>Importer un fichier</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="l_typeFile">
         <property name="text">
          <string>Type de données</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_chooseTypeFile">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <item>
          <property name="text">
           <string>- Choisir un type de donnée -</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Microsonde - Données de reférence</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>img/xlsx_.ico</normaloff>img/xlsx_.ico</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Analyses pXRF - Données de calibration</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>img/xlsx_.ico</normaloff>img/xlsx_.ico</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Analyses pXRF - Données inconnues</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>img/xlsx_.ico</normaloff>img/xlsx_.ico</iconset>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tb_browserFile">
         <property name="minimumSize">
          <size>
           <width>255</width>
           <height>35</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Parcourir ...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>img/import.ico</normaloff>img/import.ico</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
